### PR TITLE
Unreferenced formal arguments

### PR DESCRIPTION
**Pull request by cugone**
Part of splitting #155 up into smaller commits.

### CODE REVIEW

File: src/UI/Core/ComboBox.cpp [hunk 1]
```diff
@@ -115,7 +115,8 @@ void ComboBox::onMouseDown(EventHandler::MouseButton button, int x, int y)
 }
 
 
-void ComboBox::onMouseWheel(int x, int y)
+//TODO: Implement in terms of internal ListBox.
```
Comment (DanRStevens) [inline src/UI/Core/ComboBox.cpp]: This randomly added `TODO` seems out of place here. Seems like it should become a GitHub issue, or become a new commit (probably in a new PR).

File: src/UI/Core/ListBox.cpp [hunk 2]
```diff
@@ -198,8 +198,8 @@ void ListBox::dropAllItems()
 	_updateItemDisplay();
 }
 
-
-void ListBox::onMouseDown(EventHandler::MouseButton button, int x, int y)
+//TODO: Intended to only react if LMB?
```
Comment (DanRStevens) [inline src/UI/Core/ListBox.cpp]: Same, this comment should be a GitHub issue or new PR.

File: src/UI/Core/ComboBox.cpp (hunk 1 above)
Comment (cugone) [reply src/UI/Core/ComboBox.cpp]: Curious, these should have been removed. I must have missed them during one of the many rebases.

Comment (cugone) [reply src/UI/Core/ComboBox.cpp]: Fixed

File: src/UI/Core/ListBox.cpp (hunk 2 above)
Comment (cugone) [reply src/UI/Core/ListBox.cpp]: Fixed

File: src/UI/Core/ComboBox.cpp (hunk 1 above)
Comment (cugone) [reply src/UI/Core/ComboBox.cpp]: The combobox already reacts to the scroll wheel.

**Pull request merged**